scripts/handle-release.sh

```
@@ -182,8 +179,12 @@ if [[ "$IS_PR" == "true" && -n "$PR_BRANCH" ]]; then
 else
   git checkout main
 fi
+
+yarn install
 yarn workspaces foreach --all --topological --no-private run build
 
+echo "Publishing in order: ${PUBLISH_ORDER[@]}"
```

Shellcheck (0.10.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)